src/vue/backend/src/views/guided-setup/SocialProConfigure.vue

```diff
@@ -20,14 +20,14 @@ import { mapState, mapActions } from 'vuex'
 import guidedSetupMixin from '@/mixins/guidedSetup'
 import pageGuard from '@/mixins/pageGuard'
 import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
-import AgentsSettings from '@/templates/AgentsSettings.vue'
+import socialProForm from '@/templates/socialProForm.vue'
 export default {
     mixins: [
```

Comment:
You're missing the name of the file

Reply:
name added to this view and the other guided setup views.